pycc/ccresponse.py

```diff
@@ -283,6 +286,8 @@ def linresp(self, A, B, omega, e_conv=1e-13, r_conv=1e-13, maxiter=200, max_diis
                 pertkey = B + "_" + self.cart[axis]
                 X_key = pertkey + "_" + f"{omega:0.6f}"
                 print("Solving right-hand perturbed wave function for %s:" % (X_key))
+                #X_2[pertkey] = self.solve_right(self.pertbar[pertkey], omega, e_conv, r_conv, maxiter, max_diis, start_diis)
```

Comment:
Remove extraneous code and commented-out lines.

Reply:
Okay